tools/ci/fetch_ci_deps.cmake

```diff
@@ -8,7 +8,7 @@
         FetchContent_Declare(
             fwk_io
             GIT_REPOSITORY https://github.com/xmos/fwk_io.git
-            GIT_TAG        f35192c19824d60740003be5448c67f5df927014
+            GIT_TAG        dcf1a87ddd006c328906094ec12493e604bc9b2f
```

Comment:
can we use v3.3.0 instead of the hash here?

Reply:
I am not sure. Let me try..

Reply:
Yes this works fine and is much clearer - thanks